pyscriptjs/src/main.ts

```diff
@@ -99,12 +100,11 @@ class PyScriptApp {
     loadRuntime() {
         logger.info('Initializing runtime');
         if (this.config.runtimes.length == 0) {
-            showError('Fatal error: config.runtimes is empty');
-            return;
+            throw new UserError('Fatal error: config.runtimes is empty');
         }
 
         if (this.config.runtimes.length > 1) {
-            showError('Multiple runtimes are not supported yet. ' + 'Only the first will be used');
+            showWarning('Multiple runtimes are not supported yet. Only the first will be used');
```

Comment:
"funny". I think that originally I put the `+` because I split the message on two lines.
I don't know why it was put on a single line later, maybe some autoformatter?

Reply:
Oh I joined them because the banner was showing them on the same line 🤔 I'll add a <br> to split it up

Reply:
I mean, I split them in the src code only because the line was too long and it didn't fit my editor.
I think we should decide an official indication for a line width limit